src/Cake.Curl.Tests/CurlDownloadFileTests.cs

```diff
@@ -469,6 +470,76 @@ public void Should_Not_Set_The_RetryConnRefused_Option_As_Argument()
                 // Then
                 Assert.DoesNotContain("--retry-connrefused", result.Args);
             }
+
+            [Theory]
+            [MemberData(nameof(CurlTimeSpanTestData.TimeSpanData), MemberType = typeof(CurlTimeSpanTestData))]
+            public void Should_Set_The_MaxTime_Option_As_Argument(TimeSpan maxTime)
+            {
+                // Given
+                var fixture = new CurlDownloadFileFixture
+                {
+                    Settings = { MaxTime = maxTime }
+                };
+
+                // When
+                var result = fixture.Run();
+
+                // Then
+                Assert.Contains(
+                    $"--max-time {maxTime.TotalSeconds.ToString(CultureInfo.CurrentCulture)}",
```

Comment:
Here, we can simply say `maxTime.TotalSeconds` since the compiler will add the call to `ToString()` when using [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated). Also, when invoking `ToString()` on a `Double` the usage of the current culture is implicit.

Reply:
@ecampidoglio I agree, I will change this. Thanks for pointing this out.